WordPress/src/main/res/layout/media_picker_file_item.xml

```diff
@@ -60,22 +60,6 @@
 
     </RelativeLayout>
 
-    <com.google.android.material.textview.MaterialTextView
-        android:id="@+id/text_selection_count"
-        android:layout_width="@dimen/media_selection_dot"
-        android:layout_height="@dimen/media_selection_dot"
-        android:layout_marginBottom="@dimen/margin_medium"
-        android:layout_marginStart="@dimen/margin_medium"
-        android:background="@drawable/photo_picker_circle_selector"
-        android:elevation="4dp"
```

Comment:
Again just to be sure it was intentional, I see we removed the elevation in the `media_picker_selection_count.xml` (and eventually we keep the elevation it would be probably worth using a dimension for the 4dp; I'm sure you thought the same already right? 😄 )

Reply:
I've removed the elevation because (and this is really hard to notice) it was adding this weird effect of a white shape in the middle of the selector. Is there any reason why we should keep it?

Reply:
> Is there any reason why we should keep it?

Just wanted to be sure it was intentional, I see the effect you mention especially with documents items where we have a light background. I'm fine with removing it, thanks for the clarification 🙇